.github/workflows/build_osgeo4w.sh

```diff
@@ -68,7 +70,7 @@ dist_esc="$src_esc\\\\$dist"
         mswindows/osgeo4w/env.bat.tmpl
     cat <<EOT
 
-set PATH=C:\\msys64\\mingw64\\bin;%OSGEO4W_ROOT%\\apps\\msys\\bin;%PATH%
+rem set PATH=C:\\msys64\\mingw64\\bin;C:\\msys64\\usr\\bin;%PATH%
```

Comment:
```suggestion
set PATH=%PATH%;C:\msys64/usr/bin/;C:/msys64/mingw64/bin
```

MSYS is not available in OSGeo4w V2, on the PATH it needs to come after the elements of the OSGeo4W related stuff, so OSGeo4W-Python3 is found first. But bash.exe from MSYS is needed for unittests that are shell scripts...